### ISSUE TITLE
should give parser errors

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#609]
```abap
DATA foo TYPE STANDARD TABLE.
DATA foo TYPE STANDARD TABLE OF.
```
